cli/src/utils.ts

```diff
@@ -61,9 +61,25 @@ export async function isOraklNetworkApiHealthy() {
 
 export async function isOraklFetcherHealthy(url: string) {
   try {
-    return 200 === (await axios.get(url))?.status
-  } catch (e) {
-    console.error(`Orakl Network Fetcher [${url}] is down`)
+    if (!(await isValidUrl(url))) {
```

Comment:
Just suggestion. I think we should be careful with capturing more than unnecessary amount code with `try` which might lead to code that is harder to read. Below, in the suggestion I move the `try` branch below `isValidUrl` which theoretically could lead to missing exception thrown by `isValidUrl` function, however, the function does not throw anything and only returns boolean.

```typescript
export async function isOraklFetcherHealthy(url: string) {
  if (!(await isValidUrl(url))) {
    console.error('Invalid URL')
    return false
  }

  try {
    const response = await axios.get(url)
    if (response.status === 200) {
      return true
    } else {
      console.error(`Orakl Network Fetcher [${url}] is down. HTTP Status: ${response.status}`)
      return false
    }
  } catch (error) {
    console.error(
      `An error occurred while checking the Orakl Network Fetcher [${url}]: ${error.message}`
    )
    return false
  }
}
```

Reply:
I totally agree with you, I will push again after fixing this